Contribute/docfx.json

```
@@ -48,7 +47,12 @@
       "ms.topic": "contributor-guide",
       "ms.custom": "external-contributor-guide"
           },
-    "fileMetadata": {},
+    "fileMetadata": {
+      "titleSuffix":{
+        "content/**.md": "Contributor guide",
```

Usually it's/**/ which means anything under this folder

```suggestion
        "content/**/*.md": "Contributor guide",
```